CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 
 ### Fixed
 - Fix missing properties on UpdateOperation ([#744](https://github.com/opensearch-project/opensearch-java/pull/744))
+- Fix pattern replace by making flag optional as on api  ([#895](https://github.com/opensearch-project/opensearch-java/pull/895))
```

Comment:
The changelog should be in Unreleased 2.x section.

Reply:
Moved to the good section!